it/controllers/CheckYourAnswersControllerISpec.scala

```diff
@@ -121,6 +157,24 @@ class CheckYourAnswersControllerISpec extends IntegrationSpecCommonBase {
       request.header.headers(LOCATION) shouldBe controllers.routes.CheckYourAnswersController.onPageLoadForConfirmation().url
     }
 
+    "redirect the user to the confirmation page on success for fire or flood" in {
+      PenaltiesStub.successfulAppealSubmission
+      val fakeRequestWithCorrectKeys: FakeRequest[AnyContent] = FakeRequest("GET", "/check-your-answers").withSession(
```

Comment:
Could you change the "GET" to a "POST" it makes no functional difference, I missed this on my PR's :sorry: